### PR TITLE
Bumped libbuildpack-dynatrace to 1.4.1

### PR DESCRIPTION
**Pull request by arthfl**
* A short explanation of the proposed change:
Bumped libbuildpack-dynatrace to 1.4.1 to include the networkzone parameter when downloading the OneAgent from the API
* An explanation of the use cases your change solves
This includes the networkzone parameter when downloading the OneAgent from the API
* [x] I have viewed signed and have submitted the Contributor License Agreement

* [x] I have made this pull request to the `develop` branch

* [x] I have added an integration test

### CODE REVIEW

**Comment by arthfl** [comment]
Dear maintainers, any ETA on this?

File: src/php/integration/deploy_a_php_app_with_dynatrace_test.go [hunk 1]
```diff
@@ -165,4 +165,30 @@ var _ = Describe("Deploy app with", func() {
...
 		Expect(app.Stdout.String()).ToNot(ContainSubstring("Extracting Dynatrace OneAgent"))
 	})
+
+	Context("deploying a PHP app with Dynatrace agent with configured network zone", func() {
+		It("checks if networkzone setting was successful", func() {
+			serviceName := "dynatrace-" + cutlass.RandStringRunes(20) + "-service"
+			command := exec.Command("cf", "cups", serviceName, "-p", fmt.Sprintf("'{\"apitoken\":\"secretpaastoken\",\"apiurl\":\"%s\",\"environmentid\":\"envid\", \"networkzone\":\"testzone\"}'", dynatraceAPIURI))
+			_, err := command.CombinedOutput()
+			Expect(err).To(BeNil())
+			createdServices = append(createdServices, serviceName)
```
Comment (arjun024) [inline src/php/integration/deploy_a_php_app_with_dynatrace_test.go:175]: What is `createdServices` here?

**Review by arjun024 — CHANGES_REQUESTED** [review]

**Comment by arjun024** [comment]
I have made changes to your diff because (1) f-Strings are not supported in py 2.6 which is used for extensions in the php-buildpack and (2) the tests were expecting logs from libbuildpack-dynatrace instead of the extension. Please run the tests locally before submitting PRs, for quicker review by maintainers.

**Comment by arthfl** [comment]
@arjun024  Thanks a lot :-)

**Pull request merged**